qutebrowser/mainwindow/tabbedbrowser.py

```diff
@@ -190,6 +190,25 @@ def current_url(self):
         qtutils.ensure_valid(url)
         return url
 
+    def check_forms_can_be_destroyed(self, tab):
```

Comment:
You should add a docstring explaining what this does and what the tab argument and return value mean.